tooling/minimize/src/bb.rs

```diff
@@ -375,6 +375,8 @@ impl<'cx, 'tcx> FnCtxt<'cx, 'tcx> {
             return self.translate_rs_intrinsic(instance, args, destination, target, span);
         }
 
+        // I think `instance.def` will be `InstanceKind::Virtual` for trait object calls ?
```

Comment:
Correct

Reply:
So what is the point of this comment? Should it be like this?
```suggestion
        // FIXME: turn `InstanceKind::Virtual` calls into trait method calls
```